packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -267,6 +267,9 @@ const WorkflowsView = defineComponent({
 				? this.$locale.baseText('workflows.project.add')
 				: this.$locale.baseText('workflows.add');
 		},
+		templateRepositoryURL() {
```

Comment:
what's the difference here? 
when debugging `this.templatesStore` was `undefined`. won't that still be the case?

Reply:
Changing method to computed property will alter the lifecycle. It hard to reproduce this one, but the hunch is that the method is being executed as soon as it's encountered int the template which causes store not to be defined in some conditions.

Reply:
but why would all the other stores be defined, and just `templatesStore` be `undefined`?? 🤔 

not trying to block this PR, just want to make sure that you guys are certain that this fix will work. Since I still can't reproduce this locally, I can't test this.
Maybe we should create a test patch image with just this fix, and deploy to Mutasem's instance ?